concurrency/3.0/_index.md

```diff
@@ -3,38 +3,44 @@ title: "Jakarta Concurrency 3.0"
 date: 2022-04-27
 summary: "Jakarta EE 10 Release"
 ---
-Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
+<!-- Please provide a short description of the specification. -->
+<!-- Typically this will not change cfrom version to version. -->
 
+Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits. 
 
-# Plan
+<!-- Please describe the high-level changes made to Jakarta Wombat 1.0. --> 
+<!-- The intent is for the first two sections to be an executive summary in the range of 300 to 800 characters. -->
+<!-- Links can accompany the executive summary, but cannot substitute for an executive summary. -->
 
-Jakarta Concurrency 3.0 contains the following features:
-
-* Asynchronous methods
+### New Features
+This release adds the following
+* Asynchronous methods (See Chapter 5, *Asynchronous Methods*)
 * Context-aware completion stages and completable futures
 * Context propagation to parallel streams operations
-* Modernization of the Trigger mechanism and Cron support
-* Propagation of third party context types
-* Resource definition annotations
+* Modernization of the Trigger mechanism with time zone support
+* Propagation of third party context types (See Chapter 4, *Thread Context Providers*)
+* Resource definition annotations and corresponding deployment descriptor elements
+###  Removals, deprecations or backwards incompatible changes
 
+### Minimum Java SE Version
+* This release requires Java SE 11 or higher
```

Comment:
I usually just put `**11**` here. Feels a little redundant to spell it out, but I'm ok with that if it can bring more clarity. Maybe bold "Java SE 11"?